plugins/modules/jcliff.ps1

```diff
@@ -195,6 +195,12 @@ $spec = @{
                         value = @{ type = "str"}
                     }
                 }
+                logging = @{ type = "list"; required = $false ; elements = "dict";
+                    options = @{
+                        name = @{ type = "str" , required = $true }
```

Comment:
the separator between the parameters is `;` and not `,`.

If you want, feel free to merge as soon as the CI passes